src/pages/Consultations/components/Calendar.tsx

```diff
@@ -0,0 +1,54 @@
+import { getSchedule } from '@/services/escola-lms/consultations';
+import { Badge, Calendar, message } from 'antd';
+import moment from 'moment';
+import { useCallback, useEffect, useState } from 'react';
+import { FormattedMessage } from 'umi';
+
+const consultationStatus = {
+  reported: 'warning',
+  reject: 'error',
+  approved: 'success',
+};
+
+const ConsultationCalendar: React.FC<{ consultation: number }> = ({ consultation }) => {
+  const [appointments, setAppointments] = useState<API.ConsultationAppointment[]>([]);
+
+  useEffect(() => {
+    const fetch = () => {
```

Comment:
zmieniłbym nazwę na fetchConsultation czy coś w tym stylu. Fetch się myli ze zwykłym fetchem() i myslałem właśnie że to jest on